src/main/java/de/dennisguse/opentracks/util/IntentDashboardUtils.java

```diff
@@ -87,6 +101,10 @@ public static void startDashboard(Context context, boolean isRecording, Track.Id
         clipData.addItem(new ClipData.Item(uris.get(MARKERS_URI_INDEX)));
         intent.setClipData(clipData);
 
+        if ((targetPackage != null) && (targetClass != null)) {
```

Comment:
Should we print a warning if only one was set?
And a logcat output would be nice for debugging to check if actually something was received.

Reply:
Fixed